src/test/java/net/snowflake/ingest/streaming/internal/RegisterServiceTest.java

```diff
@@ -53,6 +66,42 @@ public void testRegisterServiceTimeoutException() throws Exception {
     }
   }
 
+  // Ignore since it runs for BLOB_UPLOAD_TIMEOUT_IN_SEC * BLOB_UPLOAD_MAX_RETRY_COUNT
```

Comment:
@sfc-gh-tzhang added a test which will simulate retries, but ignored since it will take > 1 minute to complete.